src/components/views/OnlineModListWithPanel.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+    <div>
+        <OnlineRowCard
+            v-for='(key, index) in pagedModList' :key="`online-${key.getFullName()}-${index}-${settings.getContext().global.expandedCards}`"
+            :image="getImageUrl(key)"
+            :id="index"
+            :is-selected="selectedMod === key"
+            @click="() => emitCardClick(key)">
+            <template v-slot:title>
+                <div>
+                    <span class="selectable">{{key.getName()}}</span>
+                    <div>
+                        <span class="card-byline">{{key.getOwner()}}</span>
+                    </div>
+                </div>
+            </template>
+            <template v-slot:other-icons>
+                <span class='card-header-icon' v-if="isModDeprecated(key)">
+                    <i class='fas fa-exclamation-triangle' v-tooltip.left="'This mod is potentially broken'"></i>
+                </span>
+                <span class='card-header-icon' v-if="key.isPinned() && !readOnly">
+                    <i class='fas fa-map-pin' v-tooltip.left="'Pinned on Thunderstore'"></i>
+                </span>
+                <span class='card-header-icon' v-if="key.getDonationLink() && !readOnly">
+                    <Link :url="key.getDonationLink()" target="external" tag="span">
+                        <i class='fas fa-heart' v-tooltip.left="'Donate to the mod author'"></i>
+                    </Link>
+                </span>
+                <span class='card-header-icon' v-if="isThunderstoreModInstalled(key) && !readOnly">
+                    <i class='fas fa-check' v-tooltip.left="'Mod already installed'"></i>
+                </span>
+            </template>
+        </OnlineRowCard>
+    </div>
+</template>
+
+<script lang="ts">
+
+import { Prop, Vue } from 'vue-property-decorator';
+import Component from 'vue-class-component';
+import ThunderstoreMod from '../../model/ThunderstoreMod';
+import ManagerSettings from '../../r2mm/manager/ManagerSettings';
+import { ExpandableCard, Link } from '../all';
+import DownloadModModal from './DownloadModModal.vue';
+import ManifestV2 from '../../model/ManifestV2';
+import DonateButton from '../../components/buttons/DonateButton.vue';
+import CdnProvider from '../../providers/generic/connection/CdnProvider';
+import { valueToReadableDate } from '../../utils/DateUtils';
+import OnlineRowCard from 'components/OnlineRowCard.vue';
+
+@Component({
+    components: {
+        OnlineRowCard,
+        DonateButton,
+        DownloadModModal,
+        ExpandableCard,
+        Link
+    }
+})
+export default class OnlineModListWithPanel extends Vue {
+
+    @Prop()
+    pagedModList!: ThunderstoreMod[];
+
+    @Prop()
+    selectedMod: ThunderstoreMod | null = null;
+
+    @Prop({default: false})
+    readOnly!: boolean;
+
+    private funkyMode: boolean = false;
```

Comment:
- This doesn't seem to be used?
- Thus `created` is unnecessary
- And since OnlineRowCards can't be expanded, `${settings.getContext().global.expandedCards}` could be removed from the key attribute
- Thus `settings` getter is unnecessary